src/routes/requestRoutes.js

```diff
@@ -30,4 +32,9 @@ app.get('/managers/:managerId', managerValid, checkToken, checkManager, checkMan
 app.patch('/:requestId/:status', singleReqValid, checkToken, checkManager, checkExistingTrip, checkLineManager, tripValidation, changeRequestStatus);
 app.post('/:requestId/comments', addCommentValidation, checkToken, checkExistingTrip, checkTripOwner, addComment);
 app.get('/:requestId/comments', viewCommentValidation, checkToken, checkExistingTrip, checkTripOwner, viewComment);
+app.get('/managers/:managerId', checkToken, checkManager, checkManagerId, getManagerRequests);
+app.patch('/:requestId/:status', checkToken, checkManager, checkExistingTrip, checkLineManager, tripValidation, changeRequestStatus);
+app.get('/notifications/manager/:managerId', checkToken, checkManager, checkManagerId, requestController.getRequestsNotification);
+app.delete('/notification/:notificationId/manager/:managerId', checkToken, checkManager, checkManagerId, checkNotificationId, requestController.deleteNotification);
```

Comment:
awesome work so far! I think as every user can receive a notification, these routes should be in userRoute. in a format like this : api/v1/users/:userId/notifications. As the manager Id is also a userId, you can pass it to that route